src/bin/casc/args.rs

```diff
@@ -8,20 +8,20 @@ use clap::Parser;
     about = "Compile Cascade SELinux policies into CIL",
     long_about = "Compile Cascade SELinux policies into CIL.
 
-The -o option to combine all policies and the -s option to build individual systems are mutually exclusive. See the OPTIONS section for more information on these options."
+The -o option to combine all policies and the -s option to build individual machines are mutually exclusive. See the OPTIONS section for more information on these options."
 )]
 pub struct Args {
     /// List of input files to process. Directories are searched recursively.
     #[clap(required(true))]
     pub input_file: Vec<String>,
     /// This is the default behavior.
-    /// Combine all policies into a monolithic system policy with system configuration options set to default values.
+    /// Combine all policies into a monolithic machine policy with machine configuration options set to default values.
     /// The generated CIL file is named OUT_FILENAME.
     #[clap(default_value = "out.cil", short, value_parser = clap::builder::ValueParser::new(parse_out_filename))]
     pub out_filename: String,
-    /// Build the systems from the SYSTEM_NAMES list. "-s all" to build all defined systems.
+    /// Build the machines from the MACHINE_NAMES list. "-s all" to build all defined machines.
```

Comment:
The short version of this will now be "-m" I imagine.  We should verify that and change the "-s" direction.